.github/azure-pipelines.yaml

```diff
@@ -0,0 +1,54 @@
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - ^v\d+\.\d+(\.\d+)?(-\S*)?$
+
+pr:
+- master
+
+resources:
+  repositories:
+  - repository: azure_template
+    type: github
+    name: mdolab/.github
+    endpoint: mdolab
+
+stages:
+- stage: Test
+  dependsOn: []
+  displayName: Test
+  jobs:
+  - job:
+    pool:
+      vmImage: 'ubuntu-20.04'
+    strategy:
+      matrix:
+        'python36':
+          PYTHON_VERSION: '3.6'
```

Comment:
Are these used anywhere? I don't see a Python action or anything, are you sure we are using these versions of Python here?

Reply:
That python should be inherited when we create the virtual environment, as the `python3` command should be linked to the specified version of python. We can add in a `python3 --version` call to double check this if you would like.

Reply:
We are actually missing this part
```
  - task: UsePythonVersion@0
    inputs:
      versionSpec: $(PYTHON_VERSION)

```

Reply:
Ah thank you! My mistake. I'll add that in.

Reply:
Yep that's what I was talking about.